sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -1432,6 +1432,9 @@ public virtual Response DeleteModel(string modelId, DeleteModelOptions options =
         /// <returns>The pageable list <see cref="AsyncPageable{T}"/> of query results.</returns>
         /// <remarks>
         /// For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
+        /// Note that there may be a delay between before changes in your instance are reflected in queries.
```

Comment:
The note seemed a little hard to understand. This might be better - 'there may be a delay between changes in your instance reflecting in queries'?

Reply:
I used the same language that is used in the official document that is being linked

Reply:
I see. You can keep it but we can provide them feedback that it is not very clear :)

Reply:
Atleast if we remove "between" from the wording, it will be easier to comprehend.

Reply:
We should make it easy for our customers to understand this even if the official documentation is not the best.

Reply:
FWIW, I don't find it confusing. Can you elaborate @vinagesh?

Reply:
Actually, I read your comments in the Java PR and now I get it. I agree, the word "between" should be removed.